scripts/start-elastic.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+docker run \
+  --name elasticsearch \
+  -d \
+  --net builds-network \
+  -p 9200:9200 \
+  -p 9300:9300 \
+  -e "discovery.type=single-node" \
+  --memory="1g" \
+  docker.elastic.co/elasticsearch/elasticsearch:7.12.1
+
+sudo docker run \
```

Comment:
Is `sudo` here just because you normally need it to run docker as root in your environment? Or does this specific image for some reason require it? That would be strange and we would like to avoid that, especially that when we move this to kubernetes the images will be started in a slightly different way

Reply:
removed